config/crd/bases/bootstrap.cluster.x-k8s.io_kubeadmconfigtemplates.yaml

```diff
@@ -16,7 +16,6 @@ spec:
     listKind: KubeadmConfigTemplateList
     plural: kubeadmconfigtemplates
     singular: kubeadmconfigtemplate
-  preserveUnknownFields: false
```

Comment:
We likely don't want to drop this, it may require updating the kubebuilder annotation for KubeadmConfigTemplate to set `crd:preserveUnknownFields=false` as per https://github.com/kubernetes-sigs/controller-tools/pull/362